### ISSUE TITLE
Internal Server Error when trying to access npm packages

### ISSUE THREAD

**Issue by zoe-codez** [go-gitea/gitea#20755]
### Description

I cannot access the details of any of the NPM packages being published from my server from the web UI. Url: `/mp3three/-/packages/npm/@steggy%2Fhome-assistant/22.33.13`

The published docker container seems fine, currently all of my npm packages are giving 500 errors. I am able to interact with them just fine via yarn (install, upgrade, etc), and there are no issues deleting the packages from the site management screen. Every time that I load the page giving me the 500 error, I get this line in the logs:

```
...s/context/context.go:232:HTML() [E] [62f40b0b] Render failed: template: package/metadata/npm:5:27: executing "package/metadata/npm" at <.PackageDescriptor.Properties>: can't evaluate field Properties in type interface {}
```

This is an an example `package.json` that was published with one of the packages:
```json
{
  "author": {
    "name": "mp3three"
  },
  "name": "@steggy/home-assistant",
  "displayName": "Home Assistant",
  "license": "MIT",
  "version": "22.33.13",
  "scripts": {
    "postinstall": "npx hass-type-generate"
  },
  "main": "./src/index.js",
  "typings": "./src/index.d.ts",
  "dependencies": {
    // ...
  },
  "peerDependencies": {}
}
```

### Gitea Version

gitea/gitea:latest docker image (1.17.0)

### Can you reproduce the bug on the Gitea demo site?

- Still waiting on demo site email to sign up

### Screenshots

![2022-08-10_15-10](https://user-images.githubusercontent.com/35661840/184010667-da998d37-3e06-472e-b354-0042a454d719.png)

### Operating System

Debian > Docker > Gitea

### How are you running Gitea?

gitea/gitea:latest as of the time of opening this issue

### Database

MySQL

**Comment by silverwind** [comment]
I think it's because package is missing a `description`, we shoul to handle its absence gracefully.

**Comment by mp3three** [comment]
I purged all my existing packages, added added a description to this one as a test, and published again. Still receiving a 500 error

```json
{
  "author": {
    "name": "mp3three"
  },
  "name": "@steggy/home-assistant",
  "displayName": "Home Assistant",
  "license": "MIT",
  "version": "22.33.13",
  "scripts": {
    "postinstall": "npx hass-type-generate"
  },
  "description": "HELLO WORLD",
  "main": "./src/index.js",
  "typings": "./src/index.d.ts",
  "dependencies": {
    // ...
  },
  "peerDependencies": {}
}
```

Is there other fields I am missing?

**Comment by silverwind** [comment]
Then my assumption is wrong. Your data looks fine. I think the minimum required properties should be just `name` and `version.`

**Comment by KN4CK3R** [comment]
Duplicate of https://github.com/go-gitea/gitea/issues/20670

But I missed to create a backport for that PR.

**Comment by silverwind** [comment]
Is it? Seems unrelated to unpublish here.

**Comment by lafriks** [comment]
I think this was fixed by #20675, it just referenced wrong issue it fixed

**Comment by KN4CK3R** [comment]
No, #20670 has the 500 error too. The unpublish was another feature request extracted from #20670

**Comment by mp3three** [comment]
I am able to confirm this issue is resolved already. Built a docker container from HEAD, and the page properly loads

Thank you all
